effectsize_exemplar_simple.Rmd

```diff
@@ -0,0 +1,127 @@
+
+### Difference in means with Student's t confidence interval
+
+While the response distributions are non-normal, the sampling distribution of the difference in means will still be defined on $(-\infty, +\infty)$ and approximately symmetrical (per the central limit theorem), so we can compute a *Student's t distribution confidence interval* for the difference in means.
+
+```{r t_test}
+t_result <- 
+  t.test(completion_time_ms ~ group, data = data) %>%
+  tidy()    # put result in tidy tabular format
+t_result
+```
+
+The `tidy()`ed output of the `t.test()` function includes an estimate of the mean difference in milliseconds (`estimate`) as well as the lower (`conf.low`) and upper (`conf.high`) bounds of the 95% confidence interval. 
+
+
+### Reporting simple effect size
+
+Ideally, we would have space in our paper to report the effect size graphically:
+
+```{r ci_plot, fig.height = 1, fig.width = 5}
+p_simple_effect_size <-   # save for the teaser figure
+  t_result %>% 
+  ggplot(aes(y = "A - B", x = estimate, xmin = conf.low, xmax = conf.high)) +
+  geom_pointrangeh() +
+  geom_vline(xintercept = 0, linetype="dashed") +
+  xlab("Mean difference in completion time (ms) with 95% CI") +
+  ylab("")
+p_simple_effect_size
+```
+
+This graphical report includes all of the [elements of an effect size report that we recommend](#effectsize_faq_how_reporting):
+
+- The direction of the difference (indicated by the label `A - B`)
+- The type of estimate reported (mean difference)
+- The type of uncertainty indicated (95% CI)
+- The units (ms)
+
+Space may not always permit the graphical report. While it can be less easy to interpret, an alternative is a textual report. **Such a report should still include all of the four elements listed above.** For example:
+
+> Group `A` had a greater mean completion time than group `B` by `r format_num(t_result$estimate, 2)` milliseconds (95% CI: [`r format_num(t_result$conf.low, 2)`, `r format_num(t_result$conf.high, 2)`]).
+
+
+### Interpreting effect size: same result, different domains = different interpretations
+
+Because simple effect sizes include units, we can use our expert judgment to interpret the report. Authors may wish to do so in order to put their result in context. Because the report above includes everything necessary for other experts to come to their own conclusion, providing our own interpretation does not prevent readers from applying their own judgment and coming to different conclusions.
+
+To illustrate the effect of domain on interpreting effect size, we will imagine two different domains that might have led to the same result reported above, and write a different interpretation of the data for each.
+
+
+#### Domain 1: Physical prototyping
```

Comment:
TODO for @mjskay: move this domain up to the beginning of this exemplar to set it up, then introduce domain 2 (below) for comparison at this point.